cmd/kube-proxy/app/server.go

```diff
@@ -685,7 +685,7 @@ func (s *ProxyServer) Run() error {
 		if max > 0 {
 			err := s.Conntracker.SetMax(max)
 			if err != nil {
-				if err != errReadOnlySysFS {
+				if errors.Is(err, errReadOnlySysFS) {
 					return err
```

Comment:
humm I am little confused why replace **!=** with **errors.Is(err, errReadOnlySysFS)** for this specific case.

Reply:
thank you for commenting
sorry my mistake
the right is ```!errors.Is(err, errReadOnlySysFS)```

addtionally 
- why i proposed this change 
 just updating error handling way upto Go1.13 , that may not affect directly now. but if there is error wrapping in the future, it works

- why just this case
cause cmd/kube-proxy pkg only have error matching in this place.
it looks too much to update every place, including many code owners place.
so i thought it was good to update one pkg as a first step